packages/fast-components-styles-msft/src/button/index.ts

```diff
@@ -71,19 +51,34 @@ function applyTransaprentBackplateStyles(): CSSRules<DesignSystem> {
         },
         "&$button__disabled": {
             borderColor: "transparent",
-            color: (config: DesignSystem): string => {
-                const designSystem: DesignSystem = withDesignSystemDefaults(config);
-
-                return disabledContrast(
-                    designSystem.contrast,
-                    designSystem.foregroundColor,
-                    designSystem.brandColor
-                );
+            color: disabledForegroundContrast(designSystem),
+            fill: disabledForegroundContrast(designSystem),
+            "& $button_beforeContent, & $button_afterContent": {
+                fill: disabledForegroundContrast(designSystem),
             },
         },
+        "& $button_beforeContent, & $button_afterContent": {
+            fill: ensureNormalBrandContrast(designSystem),
+        },
     };
 }
 
+function ensureNormalBrandContrast(designSystem: DesignSystem): string {

```

Comment:
This function already exists in color utilities

Reply:
Resolved